discourse/discourse-update.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM janx/servo
+MAINTAINER Jan Keromnes "janx@linux.com"
```

Comment:
Nit: No need to list me as a maintainer 😄 I didn't know you could have multiple maintainers, but I'll fix bugs anyway even if I'm not listed.